### ISSUE TITLE
Embedded Android views may send accessibility events before the accessibility bridge knows about them

### ISSUE THREAD

**Issue by amirh** [flutter/flutter#30068]
Right now the main problem with this is that it might result with the Android accessibility framework asking the accessibility bridge for an embedded node (we do generate the flutter<-->embedded ID mapping on the fly when an event is sent) and the bridge cannot create the node as the bounds of the embedded view are not yet known.

Right now we are not returning the node in this case.

Potential ways we could handle this better are either to communicate the bounds of the embedded view to the accessibility bridge earlier, or to delay view creation until it is in the semantics tree.
